packages/components/ovh-shell/src/client/index.ts

```diff
@@ -60,9 +60,12 @@ export function initStandaloneClientApi(
   appId: ApplicationId,
   applications: Record<string, Application>,
 ) {
-  const appConfig = applications[appId];
+  let appConfig = applications[appId];
   if (!appConfig) {
-    throw new Error(`Unknown application '${appId}'`);
+    //throw new Error(`Unknown application toto '${appId}'`);
+    // by default in local we add dedicated app
+    // if the app is not registred in 2api
+    appConfig = applications['dedicated'];
```

Comment:
**question:** This seems like a temporary change. Defaulting the app dedicated can be avoided.

Reply:
how can we avoided this ? we make this fix to run in local env